index.html

```diff
@@ -1,11 +1,582 @@
+            </p>
+          </div>
+          <div id="second" class="col-lg-4">
+            <img src="assets/img/third-medal.svg" class="medal" />
+            <h3 id="prize-title">3rd Place</h3>
+            <div class="underline"></div>
+            <p class="prize-desc">
+              • <b>$100</b> Cash Prize<br>
+              • MIT Pathfinders <b>T-Shirt</b><br>
+              • Recognition on <b>MIT</b> Site<br>
+            </p>
+          </div>
+        </div>
+        <br>
+        <br>
+        <p style="text-align: center;">In addition, every participant will receive a digital certificate from MIT Pathfinders.</p>
+      </div>
+
+    </section> End Prizes Section -->
+
+    <!-- ======= Schedule Section ======= -->
+    <section id="timeline" class="section-with-bg">
+      <div class="container" data-aos="fade-up">
+        <div class="section-header">
+          <h2>Timeline</h2>
+          <p>Here is our timeline</p>
+        </div>
+
+        <ul class="nav nav-tabs" role="tablist" data-aos="fade-up" data-aos-delay="100">
+          <li class="nav-item">
+            <a class="nav-link active" href="#day-1" role="tab" data-toggle="tab">November 28th</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#day-2" role="tab" data-toggle="tab">December 7th</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#day-3" role="tab" data-toggle="tab">December 11th</a>
+          </li>
+        </ul>
+
+        <div class="tab-content row justify-content-center" data-aos="fade-up" data-aos-delay="200">
+
+          <!-- Schdule Day 1 -->
+          <div role="tabpanel" class="col-lg-9 tab-pane fade show active" id="day-1">
+            <br>
+            <h1 class="sub-heading"><b>Essay Abstract Due!</b><br></h1>
+            <h3 class="sub-headin">Please submit your essay abstract by 11:59 PM EST on Saturday, <b>November 28th</b>. A submission form will be available soon.</h3>
+          </div>
+          <!-- End Schdule Day 1 -->
+
+          <!-- Schedule Day 2 --> 
+          <div role="tabpanel" class="col-lg-9  tab-pane fade" id="day-2">
+            <br>
+            <h1 class="sub-heading"><b>Complete Essay Due!</b><br></h1>
+            <h3 class="sub-headin">Please submit your <b>complete</b> essay by 11:59 PM EST on Monday, <b>December 7th</b>. Essays will be reviewed by the committee and published on the website by <b>December 9th</b>. A submission form will be available soon.</h3>
+          </div>
+          <!-- End Schdule Day 2 -->
+
+          <!-- Schdule Day 3 -->
+          <div role="tabpanel" class="col-lg-9  tab-pane fade" id="day-3">
+            <br>
+            <h1 class="sub-heading"><b>MIT Pathfinders Event!</b><br></h1>
+            <h3 class="sub-headin">Join us for our <b>live</b> MIT Pathfinders Event with various vaccine experts and professors. We will go over submitted essays and host a QaA with the experts!</h3>
```

Comment:
hmmm... im not sure we can claim "various vaccine experts and professors" since we dont have any signed up lol

maybe just say mit pathfinders event where we everyone's essay will be presented to a panel of professionals